tests/lib/dbus.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+DISPLAY=:0
+
+# shellcheck source=tests/lib/pkgdb.sh
+. "$TESTSLIB/pkgdb.sh"
+
+init_dbus_env(){
+    export $(cat dbus.env | xargs)
+}
+
+start_dbus_loop_unit(){
+    local executable="$1"
+
+    distro_install_package dbus-x11
+
+    dbus-launch > dbus.env
+    init_dbus_env
```

Comment:
That's fine, but it's still unnecessary to have a "init_dbus_env" which has a single line. It's much more clear for the reader to just do `export $(cat dbus.env)` instead (xargs is unnecessary too).